source/includes/_aviate-catalog-apis.md

```diff
@@ -0,0 +1,518 @@
+# Aviate Catalog APIs
+
+The Aviate catalog plugin exposes catalog APIs to create individual catalog entries such as plans, products, and pricelists without the need to manage entire catalog versions. This section documents the catalog APIs exported by the Aviate catalog plugin.
+
+## Before You Begin
+
+### Tenancy & Account-level
+
+Each of the aviate plugin endpoints below require specifying the standard Kill Bill `X-killbill-apiKey` and `X-killbill-apisecret` headers, which identify the tenant that should be used. In addition, it is possible to specify an optional `accountId` query parameter to create/access catalog entries on a per-account level. 
+
+The Kill Bill system will retrieve such entries from the plugin each time there is a need to access the catalog for a particular tenant. If there are per-account entries, only these entries will be returned for the matching account.
+
+//TODO - mention ID_TOKEN here?
```

Comment:
@pierre, before I merge, do you think anything needs to be done for this comment?